docs/reference/release-notes/7.5.asciidoc

```diff
@@ -1,7 +1,77 @@
+[[release-notes-7.5.1]]
+== {es} version 7.5.1
+
+Also see <<breaking-changes-7.5,Breaking changes in 7.5>>.
+
+[[enhancement-7.5.1]]
+[float]
+=== Enhancements
+
+Features/Watcher::
+* Log attachment generation failures {pull}50080[#50080]
+
+Network::
+* Netty4: switch to composite cumulator {pull}49478[#49478]
+
+
+
+[[bug-7.5.1]]
+[float]
+=== Bug fixes
+
+Authentication::
+* Fix iterate-from-1 bug in smart realm order {pull}49473[#49473]
+
+CRUD::
+* Do not mutate request on scripted upsert {pull}49578[#49578] (issue: {issue}48670[#48670])
+
+Cluster Coordination::
+* Make elasticsearch-node tools custom metadata-aware {pull}48390[#48390]
+
+Features/ILM+SLM::
+* Handle failure to retrieve ILM policy step better {pull}49193[#49193] (issue: {issue}49128[#49128])
+
+Features/Java High Level REST Client::
+* Support es7 node http publish_address format {pull}49279[#49279] (issue: {issue}48950[#48950])
+
+Geo::
+* Geo: Fix handling of circles in legacy geo_shape queries {pull}49410[#49410] (issue: {issue}49296[#49296])
+
+Infra/Packaging::
+* Extend systemd timeout during startup {pull}49784[#49784] (issue: {issue}49593[#49593])
+
+Machine Learning::
+* [ML] Use query in cardinality check {pull}49939[#49939]
+* Fix expired job results deletion audit message {pull}49560[#49560] (issue: {issue}49549[#49549])
+* [ML] Apply source query on data frame analytics memory estimation {pull}49527[#49527] (issues: {issue}49454[#49454], {issue}49517[#49517])
```

Comment:
```suggestion
* [ML] Apply source query on data frame analytics memory estimation {pull}49527[#49527] (issues: {issue}49454[#49454], {pull}49517[#49517])
```

Reply:
I thought it was better not to link the other PR. Happy to apply your suggestion instead if you feel otherwise.

Reply:
That works for me. Thanks @jpountz!